src/lib/sectionList/listViews/sectionListViewsConfig.ts

```diff
@@ -154,12 +154,29 @@ export const modelListViewsConfig = {
     },
     indicator: {
         columns: {
-            default: ['name', DESCRIPTORS.publicAccess, 'lastUpdated'],
+            default: [
+                'name',
+                { label: i18n.t('Indicator Type'), path: 'indicatorType' },
+                DESCRIPTORS.publicAccess,
+                'lastUpdated',
+            ],
         },
         filters: {
             default: ['indicatorType'],
         },
     },
+    indicatorType: {
+        columns: {
+            default: [
+                'name',
+                { label: i18n.t('Factor'), path: 'factor' },
+                'lastUpdated',
+            ],
+        },
```

Comment:
We need to remove `publicAccess` from available columns as well. Or the user may select it as part of the view. 
I might come up with a better way to share the non-shareable columns but for now we could just do do:

```
```suggestion
        columns: {
            default: [
                'name',
                { label: i18n.t('Factor'), path: 'factor' },
                'lastUpdated',
            ],
             available: [
                'code',
                'created',
                'createdBy',
                'href',
                'id',
                'lastUpdatedBy',
                ],
        },